config/systemjs.config.js

```diff
@@ -90,13 +122,18 @@ System.config({
       "fs": "github:jspm/nodelibs-fs@0.1.2",
       "inherits": "npm:inherits@2.0.1",
       "minimalistic-assert": "npm:minimalistic-assert@1.0.0",
+      "systemjs-json": "github:systemjs/plugin-json@0.1.0",
```

Comment:
When I did a clean install of npm and jspm modules, I don't get any sign of `systemjs-json` in my `systemjs.config.js`.

@cavis Can you do a clean install? When you do, is this `systemjs-json` still in here?